cmd/ocm-backplane/console/console_test.go

```diff
@@ -174,19 +174,19 @@ var _ = Describe("console command", func() {
 			checkCapturedCommands()
 		})
 	})
-*/
-/*	Context("when kube config is invalid", func() {
-	It("should start not console server", func() {
-		testKubeCfg.CurrentContext = "undefined-context"
-		setupConfig()
 
-		mockOcmInterface.EXPECT().GetOCMAccessToken().Return(&testToken, nil).AnyTimes()
+	Context("when kube config is invalid", func() {
+		It("should start not console server", func() {
+			testKubeCfg.CurrentContext = "undefined-context"
+			setupConfig()
+
+			mockOcmInterface.EXPECT().GetOCMAccessToken().Return(&testToken, nil).AnyTimes()
```

Comment:
You will need to add mock calls for `mockOcmInterface.EXPECT().GetPullSecret()...` and `mockOcmInterface.EXPECT().GetClusterInfoByID...` in each of the test case as they're being called in `fetchPullSecretIfNotExist()`.